foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java

```diff
@@ -32,4 +36,11 @@ public void testAll() {
         Assert.assertEquals(3, listO.size());
         Assert.assertEquals(null, ConfigUtil.getProperty("notExist"));
     }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
```

Comment:
full config teardown is io.servicecomb.serviceregistry.ArchaiusUtils.resetConfig()
if we have a test common project, can move ArchaiusUtils to it.